driver-core/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
We do not have these in Quarkus. I need to try with that. It might fix the mongo crypt issues we have in native.